lib/mayaUsd/ufe/UsdCamera.cpp

```diff
@@ -181,7 +181,13 @@ float UsdCamera::fStop() const
         stageUnits = UsdGeomGetStageMetersPerUnit(stage);
     }
 
+    
```

Comment:
In 2022 Gold we set this value in Maya without sanitizing the input and a value of 0 (USD's default value) asserts.